jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/execution/sam/SamInvokeRunnerTest.kt

```diff
@@ -58,7 +58,7 @@ class SamInvokeRunnerTest {
     @Test
     fun canDebugSupportedRuntimeHandler() {
         val runConfig =
-            createHandlerBasedRunConfiguration(project = projectRule.project, runtime = Runtime.JAVA8)
+            createHandlerBasedRunConfiguration(project = projectRule.project, runtime = Runtime.JAVA17)
```

Comment:
Should we add a new test for JAVA17 instead of replacing the old one?

Reply:
@rli Is this need to be added a new test for Java 17 ?

Reply:
let's do as an integ test since this runner isn't specific to a given runtime

Reply:
Is this integ test being specific to java17  or put a separate Pr for it ?